src/masternodes/govvariables/attributes.cpp

```diff
@@ -282,6 +312,26 @@ Res ATTRIBUTES::ProcessVariable(const std::string& key, const std::string& value
 
     auto typeKey = itype->second;
 
+    if (type == AttributeTypes::Param) {
+        if (typeId == ParamIDs::DFIP2201) {
+            if (typeKey == DFIPKeys::RewardPct ||
+                typeKey == DFIPKeys::BlockPeriod) {
```

Comment:
since this block in general is prepared for other typeIds etc. Why check here for the "other" values?
This if needs to be adapted whenever a new typeId (with new typeKeys) come along. Would be better to check here for the wanted values and return the error if typeKey is not a wanted one, right?